Makefile

```diff
@@ -0,0 +1,42 @@
+PROG_NAME=olm-mermaid-graph
+#OUTPUT_TYPE=png # or pdf
+OUTPUT_TYPE=svg
+MERMAID_TEMP_SCRIPT=mermaid.mer
+#INDEX_DB_PATH_AND_NAME?=/Users/btofel/workspace/sample-operator/test-registry.db
+#INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.community-operators
+INDEX_DB_PATH_AND_NAME?=olm_catalog_indexes/index.db.4.6.redhat-operators
+
+detected_OS := $(shell uname 2>/dev/null || echo Unknown)
+
+.PHONY: all
+all: build
+
+.PHONY: build
+build:
+	go build -o bin/$(PROG_NAME) ./
+
+.PHONY: install
+install:
+	go install
+
+run: build
+	sed 's+olm_catalog_indexes/index.db.4.6.redhat-operators+$(INDEX_DB_PATH_AND_NAME)+' sqlite3.sql > sqlite3_exec.sql
```

Comment:
sed does not work in all SO. E,g that is a problem for mac. So, since we want end-user using that I think is valid we improve the usability a little. See my other comments. IMO we need to move that for the golang code and use cobra args which make that simpler to be used.

Reply:
See : https://github.com/operator-framework/index-mermaid-graph/pull/1/files#r612848316

Reply:
Can you verify that exact sed command does not work for you under MacOS? It is working for me under MacOS, but it's possible I replaced my sed with GNU sed at some point.

And, yeah, using sed is a little clunky for this, but likely the front-end of this pipeline is changing db indexes to folder of JSON, so don't want to write too much code tied to current approach.

Reply:
I use GNU sed then that will works. However, IMO has no need to ask for someone to install/configure something aditional when we can just solve that in a way that does not leverage in nothings that is specific for an SO. In this way, we can improve its usuability.